cmd/lncli/cmd_walletunlocker.go

```diff
@@ -506,6 +507,12 @@ func unlock(ctx *cli.Context) error {
 		return err
 	}
 
+	waitUntilStatus(ctx, lnrpc.WalletState_LOCKED, time.Duration(math.MaxInt64))
```

Comment:
We shouldn't swallow the error returned here. At least it should be printed to the console.